wcag/3.3.8.md

```diff
@@ -2,4 +2,27 @@
 
 (Level AA)
 
-Placeholder
+This applies directly as written, and as described in [Intent from Understanding Success Criterion 3.3.8](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html), “the Web site” with “a Web site, screen, or application”.
+
+A [cognitive function test](https://www.w3.org/TR/wcag2ict-22/#dfn-cognitive-function-test) (such as remembering a password or solving a puzzle) is not required for any step in an authentication [process](https://www.w3.org/TR/WCAG22/#dfn-processes) unless that step provides at least one of the following:
+ 
+**Alternative**
+Another authentication method that does not rely on a cognitive function test.
+ 
+**Mechanism**
+A [mechanism](https://www.w3.org/TR/WCAG22/#dfn-mechanism) is available to assist the user in completing the cognitive function test.
+ 
+**Object Recognition**
+The cognitive function test is to recognize objects.
+ 
+**Personal Content**
+The cognitive function test is to identify [non-text content](https://www.w3.org/TR/WCAG22/#dfn-non-text-content) the user provided to a Web site, screen, or [application].
+ 
+### Note 1
```

Comment:
For notes, we currently put a `>` in front of each line that's part of the note to make it one `<blockquote>` when rendered.